### ISSUE TITLE
How to get query string parameters like in near discovery

### ISSUE THREAD

**Issue by petersalomonsen** [NearSocial/viewer#175]
In near discovery ( near.org ) I can do this:

```
return <p>hello {props.name}</p>;
```

and then use the URL https://test.near.org/peterdevgovgigs.testnet/widget/propstest?name=peter which then gives the output `hello peter`

if I try the same with near.social the query string parameter disappears from the url and is not available in `props` so the output is just `hello `. See https://test.near.social/?name=peter#/peterdevgovgigs.testnet/widget/propstest

**Comment by zavodil** [comment]
https://test.near.social/#/peterdevgovgigs.testnet/widget/propstest?name=,%20sir looks good, is it a refresh issue?

**Comment by petersalomonsen** [comment]
> https://test.near.social/#/peterdevgovgigs.testnet/widget/propstest?name=,%20sir looks good, is it a refresh issue?

no, the problem is that wallet callback puts the `transactionHashes` before the hash, and not after as you have done:

here's an example https://www.youtube.com/watch?v=xFS0Vt6LCZM

**Comment by petersalomonsen** [comment]
See the updated reproduction example and video.
